datasync/chaindatafetcher/kafka/config.go

```diff
@@ -45,7 +45,7 @@ const (
 )
 
 type KafkaConfig struct {
-	SaramaConfig         *sarama.Config // kafka client configurations.
+	SaramaConfig         *sarama.Config `json:"-"` // kafka client configurations.
```

Comment:
Is it deprecated? If so, what about adding comment this is deprecated?

Reply:
This is not deprecated. `SaramaConfig` is a library configuration and it is unmarshallable. We shows Kafka options in other fields, so it can be ingored.